Automattic/wp-calypso
# Stats: Refactoring slider props and CSS for a loader

Related to #79516

## Proposed Changes

* updating CSS to align the initial loader on the purchase page
* refactoring the values that display emoji for the purchase slider

## Testing Instructions



* Open the live branch for this PR
* Navigate to `/stats/purchase/:siteSlug?flags=stats/paid-stats`
* Verify that the initial loader is centered horizontally
* Verify that the purchase slider changes emoji when browsing values other than USD

## Pre-merge Checklist
